src/script/script.cpp

```diff
@@ -274,6 +275,43 @@ const char *GetOpName(opcodetype opcode)
     }
 }
 
+bool CheckMinimalPush(const std::vector<uint8_t> &data, opcodetype opcode)
+{
+    // Excludes OP_1NEGATE, OP_1-16 since they are by definition minimal
+    DbgAssert(0 <= opcode && opcode <= OP_PUSHDATA4, return true);
+    if (data.size() == 0)
```

Comment:
looks like you have a linker error related to dbgassert and the logprintstr inside of it.

Reply:
yeah there's a linker error for win32/64. will look into it. 

while I'm here would it be acceptable to turn the `DbgAssert` to a simple `assert` in this case?

Reply:
that would seem reasonable and an easy fix since that's how other implementations are coded but maybe   @gandrewstone has more insight.

Reply:
while waiting for @gandrewstone turned `DbgAssert` into an `assert`. We can go back to original version at any time

Reply:
Let's return true rather than assert.  This function has a lot of exposure.  And it can easily be thought of as "returns true if the passed code is legal with respect to minimal push".  So its fine to return true for opcodes where the minimal data push rule is not applicable.

Reply:
ok will do